modules/minimint-mint/src/config.rs

```diff
@@ -65,8 +68,24 @@ impl GenerateConfig for MintConfig {
                 .into_iter()
                 .map(|(amount, (pk, _, _))| (amount, pk))
                 .collect(),
+            fee_consensus: FeeConsensus::default(),
         };
 
         (mint_cfg, client_cfg)
     }
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct FeeConsensus {
+    pub coin_issuance_abs: minimint_api::Amount,
```

Comment:
Not certain if this is the right place for this (unused) field. Please double check.

Reply:
Yup that looks right.